### PR TITLE
refactor: enhance the descriptions of CLI commands, closes #7572

### LINKED ISSUE #7572: Tauri CLI 2.0 DX Improvements
> This issue has been upstreamed from https://github.com/tauri-apps/tauri-docs/issues/1373 

Issues related to https://github.com/tauri-apps/tauri-docs/pull/1365

Could we adjust a few things in the Tauri CLI to improve the DX:

- [ ] Add this (or similar) to help for `dev`:
> This command will open the WebView in development mode. It makes use of the `build.devPath` property from your `tauri.conf.json` file.
> If you have entered a command to the `build.beforeDevCommand` property, this one will be executed before the dev command.

- [ ] Add this (or similar) to help for `info`:
> `tauri info` shows a concise list of information about the environment, Rust, Node.js and their versions as well as a few relevant project configurations.

- [ ] Add better copy for `completions` command: 
> The Tauri CLI can generate shell completions for Bash, Zsh, PowerShell and Fish.

Are there any instructions after running this command on how to actually make it kick into place? If so could we include those in either the help comments or log them out so the user isn't confused on why things aren't working? @FabianLars wrote some amazing things [here](https://github.com/tauri-apps/tauri-docs/blob/25b086ed5523c2e1ac6e3d1e6a86a62e654151a0/src/content/docs/2/reference/cli.mdx#L292-L424) that I think belongs in the CLI instead. 

- [x] Switch `tauri plugin add` to `tauri add` - https://github.com/tauri-apps/tauri/pull/7744

We've talked about this on a few places and I've not heard anything against this, but open to discussion if needed. Basically let's simplify this stuff as much as possible. End users don't care so much about if it's a plugin, they care about the functionality.

### PR DESCRIPTION
**Pull request by amrbashir**
ref: https://github.com/tauri-apps/tauri/issues/7756

### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: .changes/cli-plugin-add.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'tauri-cli': 'patch:feat'
+'@tauir-apps/cli': 'patch:feat'
```
Comment (amrbashir) [inline .changes/cli-plugin-add.md]:
```suggestion
'@tauri-apps/cli': 'patch:feat'
```

**Comment by FabianLars** [comment]
I like the new descriptions but why was `plugin add` added back?

**Comment by amrbashir** [comment]
it is just an alias for, `tauri add`, and I felt like it belongs under `plugin` subcommand as well, but could be removed, it is just a suggestion.

**Comment by FabianLars** [comment]
yeah so the reason was simply that all the other plugin commands operate on a plugin level/project while the add command operates on an actual tauri app so we felt like it could be confusing.

For ref, i think this was the discord convo about this: https://discord.com/channels/616186924390023171/986184094050316358/1138159565775569018

**Comment by amrbashir** [comment]
thanks for the context, will revert it then

File: .changes/cli-plugin-add.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'tauri-cli': 'patch:feat'
+'@tauri-apps/cli': 'patch:feat'
+---
+
+Add back `tauri plugin add` command as an aliast for `tauri add` command.
```
Comment (olivierlemasle) [inline .changes/cli-plugin-add.md]: @amrbashir You reverted this change (a18a3c1c), but you did not delete the change file.

Comment (amrbashir) [reply .changes/cli-plugin-add.md]: Thank you, will delete that one as well

File: tooling/cli/src/completions.rs [hunk 3]
```diff
@@ -13,7 +13,7 @@ use std::{fs::write, path::PathBuf};
 const PKG_MANAGERS: &[&str] = &["cargo", "pnpm", "npm", "yarn"];
 
 #[derive(Debug, Clone, Parser)]
-#[clap(about = "Shell completions")]
+#[clap(about = "Generate Tauri CLI shell completions for Bash, Zsh, PowerShell and Fish")]
```
Comment (lucasfernog) [inline tooling/cli/src/completions.rs]:
```suggestion
#[clap(about = "Generate Tauri CLI shell completions for Bash, Zsh, PowerShell or Fish")]
```

File: tooling/cli/src/init.rs [hunk 4]
```diff
@@ -30,7 +30,7 @@ const TEMPLATE_DIR: Dir<'_> = include_dir!("templates/app");
 const TAURI_CONF_TEMPLATE: &str = include_str!("../templates/tauri.conf.json");
 
 #[derive(Debug, Parser)]
-#[clap(about = "Initializes a Tauri project")]
+#[clap(about = "Initialize a tauri project in an existing directory")]
```
Comment (lucasfernog) [inline tooling/cli/src/init.rs]:
```suggestion
#[clap(about = "Initialize a Tauri project in an existing directory")]
```

**Review by lucasfernog — CHANGES_REQUESTED** [review]

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**